### PR TITLE
Kb/updated api

### PR DESCRIPTION
**Pull request by kbo001**
Raising this PR to merge the RequestAPI to master. I have not made changes to the README doc as you have already made some changes.

### CODE REVIEW

File: .github/workflows/request_api.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+name: Python Request API
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  repository_dispatch:
+    types: [ deploy_success ]
```
Comment (kiran-tripathy) [inline .github/workflows/request_api.yml:13]: I brought this up yesterday as well, but still checking again -- are the deploys configured in such a way that it picks up staging URL on staging deploys and that of prod's on prod deploys?

File: RequestAPI/settings.cfg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[default]
```
Comment (kiran-tripathy) [inline RequestAPI/settings.cfg:1]: we've pushed the credentials. We can't erase this from commit history, so let's update the credentials.

Comment (kbo001) [reply RequestAPI/settings.cfg:1]: I thought I added it in the gitignore to not upload this file. I will update the creds and gitignore as well

File: RequestAPI/settings-sample.cfg [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[default]
+url = https://staging.commcarehq.org/
+password =
```
Comment (kiran-tripathy) [inline RequestAPI/settings-sample.cfg:3]: This is password of what? Could this be made more readable?

Comment (kbo001) [reply RequestAPI/settings-sample.cfg:3]: This Password is used in many of the json files as a value. I'll change it to json_password.

**Review by kiran-tripathy — APPROVED** [review]

**Pull request merged**